docs/sources/write/style-guide/word-list/index.md

```diff
@@ -132,7 +132,17 @@ Use this rather than _hold the pointer over_ or _point to_.
 <!--
 ## I
 ## J
+-->
+
 ## K
+
+### kebab case
+
+Use this to refer to the casing in which lower case words are connected by dashes or hyphens.
```

Comment:
Is this really "casing"?  This is more to do with replacing spaces with dashes to "form a multi-word identifier".

Maybe a rephrase to something more like this?
```
Use this to refer to the naming convention where spaces between words are replaced with dashes to form a multi-word identifier.
```